docs/products/postgresql/howto/pagila.rst

```diff
@@ -0,0 +1,258 @@
+    
+
+.. dropdown:: Show the actors and actresses ordered by how many movies they are featured in
+
+    .. code:: sql
+
+        select
+            actor.first_name,
+            actor.last_name,
+            count(actor.first_name) featured_count
+        from
+            actor
+        left join film_actor on
+            actor.actor_id = film_actor.actor_id
+        group by
+            actor.first_name,
+            actor.last_name
+        order by
+            featured_count desc;
+
+    .. code:: text
+
+        |first_name|last_name|featured_count|
+        |----------|---------|--------------|
+        |SUSAN     |DAVIS    |54            |
+        |GINA      |DEGENERES|42            |
+        |WALTER    |TORN     |41            |
+        |MARY      |KEITEL   |40            |
+        |MATTHEW   |CARREY   |39            |
+        |SANDRA    |KILMER   |37            |
+        |SCARLETT  |DAMON    |36            |
+        |VIVIEN    |BASINGER |35            |
+        |VAL       |BOLGER   |35            |
+        |GROUCHO   |DUNST    |35            |
+
+
+.. dropdown:: Get a list of all active customers, ordered by their first name
+
+    .. code:: sql
+
+        select
+            first_name,
+            last_name
+        from
+            customer
+        where
+            active = 1
+        order by first_name asc;
+
+    .. code:: text
+
+        |first_name|last_name|
+        |----------|---------|
+        |MARY      |SMITH    |
+        |PATRICIA  |JOHNSON  |
+        |LINDA     |WILLIAMS |
+        |BARBARA   |JONES    |
+        |ELIZABETH |BROWN    |
+        |JENNIFER  |DAVIS    |
+        |MARIA     |MILLER   |
+        |SUSAN     |WILSON   |
+        |MARGARET  |MOORE    |
+        |DOROTHY   |TAYLOR   |
+
+.. dropdown:: See who rented most DVDs – and how many times
+
+    .. code:: sql
+
+        select
+            customer.first_name,
+            customer.last_name,
+            count(customer.first_name) rentals_count 
+        from
+            customer
+        left join rental on
+            customer.customer_id = rental.customer_id
+        group by 
+            customer.first_name,
+            customer.last_name
+        order by rentals_count desc;
+
+    .. code:: text
+
+        |first_name|last_name|rentals_count|
+        |----------|---------|-------------|
+        |ELEANOR   |HUNT     |46           |
+        |KARL      |SEAL     |45           |
+        |CLARA     |SHAW     |42           |
+        |MARCIA    |DEAN     |42           |
+        |TAMMY     |SANDERS  |41           |
+        |WESLEY    |BULL     |40           |
+        |SUE       |PETERS   |40           |
+        |MARION    |SNYDER   |39           |
+        |RHONDA    |KENNEDY  |39           |
+        |TIM       |CARY     |39           |
+
+
+Ready for a challenge?
+----------------------
+After playing around with the sample queries, can you use SQL statements to answer some these questions?
```

Comment:
Should we also give the solutions on a different page? I feel this could be like the crosswords, where you know you can find the solution at the end of the book

Reply:
Done!